_includes/macros/activity-snippets.html

```diff
@@ -42,7 +42,7 @@
 
 {# ==========================================================================
 
-   _activity_snippet()
```

Comment:
Not necessarily. But if not here it should probably be mentioned as private in the code comment description.